includes/Modules/Sign_In_With_Google.php

```diff
@@ -248,7 +252,11 @@ private function render_signin_button() {
 <script>
 ( () => {
 	const parent = document.createElement( 'div' );
+<?php if ( $is_woo_commerce_login ) : // phpcs:ignore Generic.WhiteSpace.ScopeIndent.Incorrect ?>
+	document.getElementsByClassName( 'login' )[0].insertBefore( parent, document.getElementsByClassName( 'woocommerce-form-row' )[0] );
```

Comment:
Let's guard against any of these not being there at all 😄 

```suggestion
	if ( document.getElementsByClassName( 'woocommerce-form-row' )[0] ) {
		document.getElementsByClassName( 'login' )[0]?.insertBefore( parent, document.getElementsByClassName( 'woocommerce-form-row' )[0] );
	}
```